### PR TITLE
Add clarification text for average times

### PR DESCRIPTION
**Pull request by steenburgh**
A few people were confused about what the times meant. What do you think of this as a potential solution to the problem? @TheBox193

screen shot 2016-07-27 at 7 30 32 pm

### CODE REVIEW

**Comment by TheBox193** [comment]
Maybe just flip the time and "Average Time" label. Might read better being "Average Time 12:05". But yeah looks cool to me 👍

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .idea
 .DS_Store
 
-*.pem
\ No newline at end of file
+*.pem
+node_modules
```
Comment (TheBox193) [inline .gitignore:5]: 👍 On this too

**Comment by steenburgh** [comment]
I swapped the label position. Here's what it looks like now:  

screen shot 2016-07-28 at 10 49 06 pm

**Comment by TheBox193** [comment]
Nice!

**Pull request merged**